ddtrace/opentracer/tracer_test.go

```diff
@@ -23,3 +27,21 @@ func TestStart(t *testing.T) {
 	assert.True(ok)
 	assert.Equal(ott.Tracer, dd)
 }
+
+func TestSpanWithContext(t *testing.T) {
+	assert := assert.New(t)
+	ot := New()
+	_, ok := internal.GetGlobalTracer().(ddtrace.Tracer)
+	assert.True(ok)
+	ott, ok := ot.(*opentracer)
+	assert.True(ok)
+
+	// start span
```

Comment:
Let's remove all inline comments in the test please. It's simple enough to be clear what is going on.